radar/infrastructure_ci_cd/gce.yaml

```diff
@@ -0,0 +1,29 @@
+name: Google Compute Engine
+logo: https://upload.wikimedia.org/wikipedia/commons/thumb/d/d8/Google-Compute-Engine-Logo.svg/2560px-Google-Compute-Engine-Logo.svg.png
+blip:
+  - date: 2019-10-14
+    ring: HOLD
+description: |
+  Google Compute Engine (GCE) is the Infrastructure as a Service component of Google Cloud Platform.
+  Google Compute Engine enables users to launch virtual machines (VMs) on demand.
+rationale: |
+  We should avoid building software that depends on virtual machines.
+  It is ok to launche VMs to perform one off tasks but shouldn't be part of the final product.
```

Comment:
Wording/typo - suggested change:
It is OK to launch VMs to perform one off tasks, but it shouldn't be part of the final product.

Reply:
Great catch! Fixed it.